fix the issue of extension causing problems in some webs

Anishali2 opened the pull request:
@paradoxe35 I've figure out that in manifest.json file we are passing styles to the web which maybe causing an problem and once I removed that it is not making any issues in web any more.

If possible can you please check that if that issue happens in your side or not so we can send this PR for approval to merge 

BEFORE::

https://github.com/user-attachments/assets/4c46572f-63fd-4e5e-98a8-49135dd5a7af

AFTER::
![image](https://github.com/user-attachments/assets/a9da12ad-3fdd-4989-b29f-6e34629f3b54)

![image](https://github.com/user-attachments/assets/01edaa51-3655-4b96-a227-f820b1bcb179)

evereq commented:
@Anishali2 so how extension keep working without styles? Can you demo extension (video) after you removed styles?

Anishali2 commented:
@evereq Sir it's not that I removed those styles from the extension or extension portal I just remove that styles to not pass to that open websites 

```
 "web_accessible_resources": [
    {
      "resources": [
        "*.html",
        "images/*",
        "fonts/*",
        "styles/*", // this is removed
        "*/install.html",
        "firebase/*.js"
      ],
      "matches": ["<all_urls>"]
    }
  ],

```
So I just removed those styles go from extension to that opened webpage 

![image](https://github.com/user-attachments/assets/1b92d905-48c4-4ee4-a5c3-0309851a4af2)

evereq commented:
@paradoxe35 thoughts on above comments, is that fine to merge PR in your opinion?

paradoxe35 commented:
@Anishali2 
I just tested the extensions. I'm still experiencing the same issue.

evereq commented:
> The popup and other pages are still working with the styles

I don't get this... Without styles or with styles? Because in this PR he EXCLUDED styles, so I don't get how everything keep working if we excluding those... MY assumption that maybe @Anishali2 tested only main popup of extension, not any other pages etc...

Anishali2 commented:
As you see Sir @evereq in this video everything is working just fine and didn't have any css conflict in any websites 

BEFORE:

https://github.com/user-attachments/assets/4c46572f-63fd-4e5e-98a8-49135dd5a7af

AFTER:

https://github.com/user-attachments/assets/88a83a2f-222e-466e-8fc5-38e1c3852b0e


@paradoxe35 when i fix this issue it didn't instantly reflect those changes maybe due to cache I re-install the extension to see the results.

paradoxe35 commented:
I believe the reason it's still working is that the styles folder isn't generated during the build. 
Therefore, I don't think this PR resolves the issue of CSS conflicts with other websites' styles.

@evereq  / @Anishali2

paradoxe35 commented:
Anyway, let's have a quick meeting so we can go through this.
@Anishali2

Anishali2 commented:
@paradoxe35 @evereq I've figure out the root cause and Yes Sir @evereq you are right the `"styles/*"` is not causing the problem but the main issue which is causing the problem is tailwind css running outside of the `rec` class boundry 


[Apex_1740187369815.webm](https://github.com/user-attachments/assets/ebe812ef-fa9a-4d46-b414-cc51d14d5e36)

paradoxe35 reviewed (approved):
I just tested it, and it looks fine to me.

@Anishali2 / @evereq
